### PR TITLE
Parser improvement for lp files

### PR DESCRIPTION
**Pull request by lvenerosy**
Slightly modified Reformat.js to be able to load files from lp_solve : 
- the comment lines containing "int" in them are now ignored
- if there is a colon in a constraint line, everything up to it is ignored

### CODE REVIEW

File: src/Reformat.js [hunk 1]
```diff
@@ -20,18 +20,22 @@
  *          work with
  **************************************************************/
 function to_JSON(input){
-    var rxo = {
+    var rxo = {//^(\-|\+){0,1}\w{1,}(?!:)
```
Comment (bchevalier) [inline src/Reformat.js]: What is this comment for?

File: src/Reformat.js [hunk 2]
```diff
@@ -20,18 +20,22 @@
...
+        "is_int": /^(?!\/\*)\W{0,}int/i,
+        "constraint_element": /(\-|\+){0,1}\w{0,}/i,
         "is_constraint": /(\>|\<){0,}\=/i,
         "is_unrestricted": /^\S{0,}unrestricted/i,
         "parse_lhs":  /(\-|\+){0,1}\s{0,1}\d{0,}\.{0,}\d{0,}\s{0,}[A-Za-z]\S{0,}/gi,
         "parse_rhs": /(\-|\+){0,1}\d{1,}\.{0,}\d{0,}\W{0,}\;{0,1}$/i,
         "parse_dir": /(\>|\<){0,}\=/gi,
         "parse_int": /[^\s|^\,]+/gi,
-        "get_num": /(\-|\+){0,1}(\W|^)\d+\.{0,}\d{0,}/g,
+        "get_num": /(\-|\+){0,1}(\W|^)\d+\.{0,1}\d{0,}/g, // Why accepting character \W before the first digit?
```
Comment (bchevalier) [inline src/Reformat.js]: @JWally , we were wondering what the ```\W``` was for? Does it means that if a character (other than an alphabet or a number) appears between the sign and the first digit, we still consider the word as a number? In which case does it happen?

**Comment by JWally** [comment]
Sorry I've been absentee on this.
For what it's worth, I built this piece of the library as a quick and dirty add on to build out LPs for LP_Solve to look for options arbitrage opportunities (this is where spy_spy_spy comes from). I probably shouldn't have let it out in the wild without doing a lot more testing on it.

I'd love to have this functionality, but its untested at the moment. Do you think it would make sense to remove it off the master branch until it can get some time? Thoughts?

**Comment by bchevalier** [comment]
@JWally , what is the functionality you are talking about? ```to_JSON``` or ```from_JSON```?
With this PR it seems that ```to_JSON``` works properly (we haven't done extensive testing though).

**Comment by JWally** [comment]
That's great!
Never mind then :-)

**Pull request merged**